pkg/controller/builder/pod_builder_test.go

```diff
@@ -34,9 +34,18 @@ func TestBuildActiveGatePodSpecs(t *testing.T) {
 		assert.NotNil(t, specs)
 		assert.Equal(t, 1, len(specs.Containers))
 		assert.Equal(t, serviceAccountName, specs.ServiceAccountName)
-		assert.Equal(t,
-			*resource.NewScaledQuantity(1, -1),
-			activeGateSpec.KubernetesMonitoringSpec.Resources.Requests[corev1.ResourceCPU])
+		assert.True(t,
+			activeGateSpec.KubernetesMonitoringSpec.Resources.Requests[corev1.ResourceCPU].Equal(
+				*resource.NewScaledQuantity(1, -1)))
```

Comment:
You can use the predefined type "resource" here - see
https://github.com/Dynatrace/dynatrace-oneagent-operator/commit/faee78b07dc8ffe124e793a73699150313900835#diff-6404509afd56daa425812290afd70394d934e45a776dcdb36087716a660c113aL204